### ISSUE TITLE
Inquiry about Hyperparameters for Reproducing Experiments

### ISSUE THREAD

**Issue by HuCutie** [sejong-rcv/INSANet#4]
Hi, I am currently working on reproducing the experiments from your paper and your work. And I would like to ensure that my implementation closely matches the results reported. Could you please provide details on the hyperparameters used during the training process? I am using the recommended software environment and default parameters provided in the code. I have trained and tested the model on both single and dual A100 GPUs, but unfortunately, I am unable to achieve the results reported in your paper.

Specifically, I am interested in information regarding:

Learning rate
Batch size
Number of epochs
Optimizer type and any associated parameters
Any other relevant hyperparameters or settings
Your assistance would be greatly appreciated, and it will be very helpful for the accuracy of my replication efforts.

Thank you very much for your time and support.

**Comment by childult-programmer** [comment]
Thanks for your interest and sorry for late reply.

Please see [config.py](https://github.com/sejong-rcv/INSANet/blob/main/src/config.py) for all hyperparameters.

Also, performance reported in paper is not the last (40) epoch, so if all settings are the same, we recommend evaluating each epoch to see the results.
